config/environments/development.rb

```diff
@@ -45,5 +45,5 @@
       password: ENV['SMTP_SETTINGS_PASSWORD']
   }
 
-  config.action_mailer.default_url_options = { host: '127.0.0.2:3000', port: 3000 }
+  config.action_mailer.default_url_options = { host: '127.0.0.1:3000', port: 3000 }
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.